lumber-generate.js

```diff
@@ -18,14 +18,23 @@ function isDirectoryExist(path) {
   }
 }
 
+let projectName;
+
 program
+  .arguments('[projectName]')
   .description('Generate the back office of your web application based on the database schema.')
   .option('-c, --connection-url', 'Enter the database credentials with a connection URL')
   .option('--no-db', 'Use Lumber without a database.')
+  .action((projectNameArg) => { projectName = projectNameArg; })
   .parse(process.argv);
 
 (async () => {
-  const config = await Prompter(program, [
+  if (!authenticator.getAuthToken()) {
+    logger.error('💀  Oops, you need to be logged in to execute this command. 💀 Try the "lumber login" command.');
+    process.exit(1);
```

Comment:
As a new user, I am stuck here and cannot do anything with the CLI, unless I create a user using ForestAdmin onboarding.